### PR TITLE
[GEOS-7347] Allow to enter an allowed area in Geofence UI

### PR DESCRIPTION
**Pull request by nmco**
https://osgeo-org.atlassian.net/browse/GEOS-7348

### CODE REVIEW

File: src/community/geofence-server/src/main/resources/GeoServerApplication.properties [hunk 1]
```diff
@@ -41,4 +41,5 @@ GeofenceRulePage.access.nullValid=*
 GeofenceRulePage.ALLOW=ALLOW
 GeofenceRulePage.DENY=DENY
 GeofenceRulePage.LIMIT=LIMIT
-GeofenceRulePage.duplicate=An identical rule is already present. Duplicates are not allowed.
\ No newline at end of file
+GeofenceRulePage.duplicate=An identical rule is already present. Duplicates are not allowed.
+GeofenceRulePage.allowedArea=Allowed area
```
Comment (aaime) [inline src/community/geofence-server/src/main/resources/GeoServerApplication.properties]: "Allowed area (WKT)" is probably going to be clearer (at least until we have a visual editor for it)

File: src/community/geofence-server/src/main/java/org/geoserver/geofence/web/GeofenceRulePage.java [hunk 2]
```diff
@@ -249,10 +343,10 @@ public void onSubmit() {
      * Makes sure we see translated text, by the raw name is used for the model
      */
     protected class GrantTypeRenderer implements IChoiceRenderer<GrantType> {
-		private static final long serialVersionUID = -7478943956804313995L;
+        private static final long serialVersionUID = -7478943956804313995L;
 
-		public Object getDisplayValue(GrantType object) {
-            return (String) new ParamResourceModel( object.name(), getPage())
+        public Object getDisplayValue(GrantType object) {
```
Comment (aaime) [inline src/community/geofence-server/src/main/java/org/geoserver/geofence/web/GeofenceRulePage.java]: Eek tabs, the GeoServer coding conventions demand spaces, have you read and applied:
http://docs.geotools.org/latest/developer/conventions/code/style.html#use-of-formatting-tools

Ah, but you use Idea right? Make it use spaces, and then maybe ask on the ml if anyone can share a Idea formatter template, I believe there are others using that IDE

**Comment by aaime** [comment]
Hi @nmco it's quite difficult to review this one, there is a lot of reformats, can you get rid of them?

File: src/community/geofence-server/src/main/java/org/geoserver/geofence/web/GeofenceRulePage.java (hunk 2 above)
Comment (nmco) [inline src/community/geofence-server/src/main/java/org/geoserver/geofence/web/GeofenceRulePage.java]: I'm the one using spaces ... the previous code used tabs.

**Comment by nmco** [comment]
Hi Andrea, sorry for the reformats I have get ride of them and I hope it help with the review. If have read and applied GeoServer contributions guidelines to my IDEA the existing format was using tabs and other strange style options. After this pull request I will probably do another one fixing the style issues.

**Pull request merged**